metadata-db-app/src/migrations/052_add_coll_ids_catalog_acls.clj

```diff
@@ -11,14 +11,14 @@
   "For a given provider and entry-titles in collection-identifier, returns list of collection concept ids"
   [provider-id collection-identifier]
   (when-let [entry-titles (get collection-identifier :entry-titles)]
-      (let [provider (-> (h/get-provider provider-id)
-                         providers/dbresult->provider)
-            t (h/get-provider-collection-tablename provider)]
-        (for [entry-title entry-titles
-              result (h/query [(format "select concept_id from metadata_db.%s where
-                                        entry_title = ?" t)
-                               entry-title])]
-          (:concept_id result)))))
+    (let [provider (-> (h/get-provider provider-id)
+                       providers/dbresult->provider)
+          t (h/get-provider-collection-tablename provider)]
+      (for [entry-title entry-titles
+            result (j/query (config/db) [(format "select concept_id from metadata_db.%s where
```

Comment:
The wrong namespace's query was getting called? Completely optional, depending on how pressing this is and how you guys usually handle bug-fixes in tests, but I'd be +1 for adding a test that duplicated the error, but with the fix in place, passes the test. Maybe check the format? Not sure how `h/query` and `j/query` differ ...